pkg/zson/raw.go

```diff
@@ -23,8 +23,8 @@ func EncodeZvals(d *Descriptor, vals []zval.Encoding) (zval.Encoding, error) {
 	return raw, nil
 }
 
-func NewRawAndTsFromZeekTSV(d *Descriptor, path []byte, data []byte) (zval.Encoding, nano.Ts, error) {
-	builder := zval.NewBuilder()
+func NewRawAndTsFromZeekTSV(builder *zval.Builder, d *Descriptor, path []byte, data []byte) (zval.Encoding, nano.Ts, error) {
```

Comment:
I think this method should be moved to zeek/parser.go and made private.

Reply:
That would break downstream packages that might depend on having it public